dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/java2sec/PermissionManager.java

```diff
@@ -352,7 +370,7 @@ private CodeSource createCodeSource(String codeBase) {
     private ProtectionDomain createProtectionDomain(CodeSource codeSource, ArrayList<Permission> permissions) {
         PermissionCollection perms = new Permissions();
 
-        if (!java2SecurityEnabled()) {
+        if (!java2SecurityEnabled) {
```

Comment:
This means that if security is not enabled from the start then classes defined by our class loaders will always have AllPermissions.  I don't think this is an issue because if some other party enables a security manager later I think it would be precarious for us to support them.

Reply:
Since I don't process any of the policy any longer in the activate, to try to support them would result in no permissions and all checks failing.  The knowledge center states that if you change permissions it is not dynamic and you need to restart the server.  I am applying that also to setting the SecurityManager.  The knowledge center says to enable it by setting a bootstrap.property to enable it.  As such I think that it should be safe to skip the policy processing when the security manager is not enabled at server startup.